packages/vsce/src/commands/clearResourceFilterCommand.ts

```diff
@@ -63,9 +63,7 @@ export function getClearResourceFilterCommand(tree: CICSTree, treeview: TreeView
           cancellable: false,
         },
         async (_, token) => {
-          token.onCancellationRequested(() => {
-            console.log("Cancelling the loading of resources");
-          });
+          token.onCancellationRequested(() => { });
```

Comment:
Have removed the unecessary console logs from these methods, but kept the empty method for if we want to add a cancellation action - am willing to debate whether this is a good idea or not